draft-gpew-priv-ppm.md

```diff
@@ -958,8 +1089,91 @@ responds with HTTP status 202 Accepted. The response MAY include a Retry-After
 header field to suggest a pulling interval to the collector.
 
 Once all the necessary reports have been prepared, the leader obtains the
-helper's encrypted aggregate share for the batch interval by sending an
-`AggregateShareReq` to the helper as described in {{aggregate-share-request}}.
+helper's encrypted aggregate share for the batch interval by requesting an
+aggregate share from each helper.
+
+To do this, the leader first computes a checksum over the set of output shares
+included in the batch window. The checksum is computed by taking the SHA256 hash of
+each nonce from the client reports included in the aggregation, then combining the
+hash values with a bitwise-XOR operation.
```

Comment:
This is identical to the definition of `checksum`, a couple paragraphs later. We should delete one.